agent/api/task/task_linux.go

```diff
@@ -26,15 +26,24 @@ import (
 	resourcestatus "github.com/aws/amazon-ecs-agent/agent/taskresource/status"
 	resourcetype "github.com/aws/amazon-ecs-agent/agent/taskresource/types"
 	"github.com/cihub/seelog"
-	containerSDK "github.com/docker/docker/api/types/container"
-	"github.com/opencontainers/runtime-spec/specs-go"
+	dockercontainer "github.com/docker/docker/api/types/container"
+	specs "github.com/opencontainers/runtime-spec/specs-go"
 	"github.com/pkg/errors"
 )
 
 const (
-	//memorySwappinessDefault is the expected default value for this platform. This is used in task_windows.go
-	//and is maintained here for unix default. Also used for testing
-	memorySwappinessDefault = 0
+	emptyVolumeName1                  = "empty-volume-1"
```

Comment:
Some of these are removed (see https://github.com/aws/amazon-ecs-agent/commit/327658d977e8ff03f11fd5c1ee7c590aab8c3567). You need to merge dev into moby branch again.

Reply:
https://github.com/aws/amazon-ecs-agent/pull/1533